app/src/main/java/com/fstyle/structure_android/screen/main/MainActivity.java

```diff
@@ -4,6 +4,8 @@
 import android.os.Parcelable;
 import android.support.annotation.NonNull;
 import android.support.design.widget.TextInputLayout;
+import android.text.InputType;
+import android.util.Log;
```

Comment:
ERROR: Unused import: android.util.Log. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports